wagtail_wordpress_import/test/tests/test_wordpress_item.py

```diff
@@ -195,3 +197,76 @@ def test_page_has_no_categories(self):
     def test_categories_count_is_zero(self):
         count = Category.objects.count()
         self.assertEqual(count, 0)
+
+
+IMPORTER_RUN_PARAMS_TEST_OVERRIDE_1 = {
+    "app_for_pages": "example",
+    "model_for_pages": "TestPage",
+    "parent_id": "2",
+    "page_types": ["post"],
+    "page_statuses": ["publish"],
+}
+
+
+@override_settings(
+    WAGTAIL_WORDPRESS_IMPORT_YOAST_PLUGIN_ENABLED=True,
+)
+class WordpressImporterTestsYoastEnabledSingleKey(TestCase):
```

Comment:
**Suggestion**: Rename this to `WordpressImporterTestsYoastMetaDescriptions`, as it is not only testing single keys.

Reply:
Thank Nick I change the naming here.